### PR TITLE
I simplified.

### PR DESCRIPTION
**Pull request by kazup01**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,45 +5,12 @@
   <br>
   <br>
 </h1>
-<h4 align="center">Note-taking app for developers. </h4>
+<h4 align="center">Note-taking app for programmers. </h4>
 <h5 align="center">macOS, Windows and Linux</h5>
 <h5 align="center">Built with Electron, React + Redux, Webpack and CSSModules</h5>
-<h1> </h1>
-
-[![Build Status](https://travis-ci.org/BoostIO/Boostnote.svg?branch=master)](https://travis-ci.org/BoostIO/Boostnote)
```
Comment (sota1235) [inline readme.md]: This line is proof that it is testing properly. Please do not remove it:sob:

File: readme.md [hunk 2]
```diff
@@ -52,15 +19,15 @@ Products are shipped worldwide. Powered by [Paintory](https://paintory.com/)
...
 
 ## Contributors
-- [dojineko](https://github.com/dojineko)
-- [Romain Bazile](https://github.com/gromain)
-- [Bruno Paz](https://github.com/brpaz)
-- [Fabian Mueller](https://github.com/dotcs)
-- [Yoshihisa Mochihara](https://github.com/yosmoc)
-- [Mike Resoli](https://github.com/mikeres0)
-- [tjado](https://github.com/tejado)
+[Great contributors](https://github.com/BoostIO/Boostnote/graphs/contributors) :tada:
```
Comment (sota1235) [inline readme.md]: It's good idea!

File: readme.md (hunk 1 above)
Comment (kazup01) [inline readme.md]: Oh,sorry! OK😚

Comment (sota1235) [inline readme.md]: I fixed it.

**Pull request merged**